Lib/unittest/mock.py

```diff
@@ -337,13 +337,19 @@ def __contains__(self, value):
 
         for i in range(0, len_self - len_value + 1):
             sub_list = self[i:i+len_value]
-            if sub_list == value:
+            if value == sub_list:
```

Comment:
I think this is not needed since order doesn't matter with comparing both ways below in `__eq__` using `(self_params == other_params or other_params == self_params)`.

Reply:
Yup, I should have removed this once the other fixes were working.